### PR TITLE
Spelling fixes. 2017-05-06

### PR DESCRIPTION
**Pull request by CraigCraig**

### CODE REVIEW

File: src/openrct2/ride/ride.h [hunk 1]
```diff
@@ -759,7 +759,7 @@ enum {
 	RIDE_TYPE_FLAG_HAS_DATA_LOGGING = 1 << 9,
 	RIDE_TYPE_FLAG_HAS_DROPS = 1 << 10,
 	RIDE_TYPE_FLAG_NO_TEST_MODE = 1 << 11,
-	RIDE_TYPE_FLAG_TRACK_ELEMENTS_HAVE_TWO_VARIETIES = 1 << 12,	// used by rides with two variaties, like the u and o shapes of the dinghy slide and the dry and submerged track of the water coaster
+	RIDE_TYPE_FLAG_TRACK_ELEMENTS_HAVE_TWO_VARIETIES = 1 << 12,	// used by rides with two variates, like the u and o shapes of the dinghy slide and the dry and submerged track of the water coaster
```
Comment (marijnvdwerf) [inline src/openrct2/ride/ride.h]: 'varieties'?

File: src/openrct2/ride/station.c [hunk 2]
```diff
@@ -84,7 +84,7 @@ static void ride_update_station_blocksection(rct_ride *ride, sint32 stationIndex
 static void ride_update_station_bumpercar(rct_ride *ride, sint32 stationIndex)
 {
 	// Change of station depart flag should really call invalidate_station_start
-	// but since bumpercars do not have station lights there is no point.
+	// but since bumper cars do not have station lights there is no point.
```
Comment (marijnvdwerf) [inline src/openrct2/ride/station.c]: @IntelOrca Isn't this supposed to be 'dodgems'?

Comment (IntelOrca) [inline src/openrct2/ride/station.c]: yes

File: src/openrct2/ride/ride.h (hunk 1 above)
Comment (CraigCraig) [inline src/openrct2/ride/ride.h]: I need to fix this lol, you're correct. I clicked the wrong word when making the correction. Thank you.

File: src/openrct2/core/Stopwatch.hpp [hunk 3]
```diff
@@ -1,52 +0,0 @@
-#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```
Comment (rwjuk) [inline src/openrct2/core/Stopwatch.hpp]: Uhh... did you mean to remove this?

Comment (CraigCraig) [inline src/openrct2/core/Stopwatch.hpp]: I had a conflict with the file, I checked the original source and it doesn't exist anymore.

Comment (rwjuk) [inline src/openrct2/core/Stopwatch.hpp]: Yep, entirely right, my mistake!!

**Comment by Gymnasiast** [comment]
This will need to be rebased and squashed (we'll do the latter when merging), but otherwise this looks good to me.

**Comment by Gymnasiast** [comment]
I squashed and rebased for you, as it looked as if it caused a bit of trouble for you.

**Pull request merged**